### DISCUSSION TITLE
[Package Source mapping] Unable to update setting since it is in an uneditable config file

### DISCUSSION THREAD

**Discussion by kartheekp-ms** [NuGet/Home#11528] [General]
@trejjam - I work on the NuGet team and can't reproduce the scenario you have mentioned https://devblogs.microsoft.com/nuget/introducing-package-source-mapping/#comment-159 here. It is possible that the scenario you have mentioned is a bug or a configuration error. AFAIK package source mapping feature wasn't backported to `net 5`

Can you please create an issue with repro steps and a sample project [here](https://github.com/NuGet/Home/issues/new?assignees=&labels=Type%3ABug%2CTriage%3AUntriaged&template=BUG.yml&title=%5BBug%5D%3A+)?

Anyone in the community experiencing the following issue?

`I configured packageSourceMapping and it works as expected, it seems it was backported to the .Net 5. I found an issue with a project using git submodules. The root and a child solution have both the nuget.config with packageSourceMapping (they share some package sources). When I exec dotnet restore it wail with the following error C:\Program Files\dotnet\sdk\5.0.402\NuGet.targets(131,5): error : Unable to update setting since it is in an uneditable config file. [...sln]. When I delete packageSourceMapping from the child solution it works again.`

**Comment by trejjam** [comment]
Hello @kartheekp-ms,
thank you for reaching me. I tried it again with net6.0.1, and it seems to work. I will check it on my windows machine asap (probable end of the week).

About the error message, I do not know why it fails. It seems like the `packageSourceMapping` affects net 5 in some way. Or there was some dependencies mix (dotnet & Visual Studio?). I do not know

**Reply by trejjam** [reply]
Hello @kartheekp-ms,
once more thank you for trying to solve the issue. I tried it again on a Windows machine with net6.0 and the issue does not occur.

Have a nice day and weekend.

**Reply by kartheekp-ms** [reply]
Thank you @trejjam for taking the time to reproduce the issue again. Glad to hear that issue does not repro.
